assets/tunes/Jericho.ts

```diff
@@ -0,0 +1,73 @@
+import { repeat } from "../../src/tuneHelper";
+export default {
+    categories: ["new", "tricky"],
+    speed: 120,
```

Comment:
This feels a pretty slow target speed if we're basing it on the Sidney Bechet version. That's more like 150. Are you happy with this?

Reply:
As we don't expect bands to play tunes as fast as the record, I originally went for 140 as a compromise. But in the end, as all the other tunes are at 100 (or 100 equivalent with Bhangra), I lowered it to match because I'd argue this is a kind of audio notation to learn the songs from, not the target speed or sound.

So basically yes, I feel like it's fine as a learning speed.